ccm_web/server/src/migrations/2021.06.30T19.52.46.add-canvas-token-expires-at.ts

```diff
@@ -0,0 +1,19 @@
+import Sequelize, { DataTypes } from 'sequelize'
+import { Migration } from '../migrator'
+
+export const up: Migration = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().addColumn(
+    'canvas_token',
+    'expires_at',
+    {
+      allowNull: false,
+      type: DataTypes.DATE,
+      field: 'expires_at',
+      defaultValue: Sequelize.fn('NOW')
```

Comment:
The default value is the current time. This `sequelize` function seems to handle the MySQL conversion. I saw other patterns online using a `sequelize.NOW` function, but that pattern didn't seem to work for me.